### PR TITLE
(PT-BR) Translating 3  files from "tsconfig"

### PR DESCRIPTION
**Pull request by juninhogomes**
Hello @khaosdoctor, @alvarocamillont, @danilofuchs e @orta

This commit is about the translation of these files:  

- forceConsistentCasingInFileNames.md
- generateCpuProfile.md
- importHelpers.md

Some notes about:  

- In the file: `forceConsistentCasingInFileNames.mds`, 
    - I used "Mantenha" (keep) instead of "Force". I did this to sound less aggressive in Portuguese but can change without problems.

- In the file: `importHelpers.md`
    - Kept in English the terms: array, async, and runtime.

And that's it. Anything, just let me know them I fix it.
Thank you guys!

### CODE REVIEW

**Comment by khaosdoctor** [comment]
I'll be reviewing now! Thanks a lot for the PR!

File: packages/tsconfig-reference/copy/pt/options/forceConsistentCasingInFileNames.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+display: "Mantenha A Consistência Nos Nomes Do Arquivo"
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/forceConsistentCasingInFileNames.md]:
```suggestion
display: "Manter a consistência nos Nomes do arquivo"
```

The display names don't have to be title-cased

File: packages/tsconfig-reference/copy/pt/options/forceConsistentCasingInFileNames.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+oneline: "Certifique-se de verificar se as letras maiúsculas e minúsculas estão corretas nas importações"
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/forceConsistentCasingInFileNames.md]:
```suggestion
oneline: "Verifica se as letras maiúsculas e minúsculas estão corretas nas importações"
```

File: packages/tsconfig-reference/copy/pt/options/generateCpuProfile.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+display: "Gerar Perfil de CPU"
+oneline: "Emita um perfil de CPU v8 do compilador executado para depuração"
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/generateCpuProfile.md]:
```suggestion
oneline: "Emite um perfil de CPU do compilador para depuração"
```

File: packages/tsconfig-reference/copy/pt/options/generateCpuProfile.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+---
+Essa opção dá a oportunidade para que o TypeScript emita um perfil de CPU v8 durante a depuração. O perfil de CPU pode oferecer informações sobre o porquê das suas compilações estarem lentas.
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/generateCpuProfile.md]:
```suggestion
Essa opção permite que o TypeScript emita um perfil de CPU durante a depuração. O perfil de CPU pode oferecer informações sobre o porquê das suas compilações estarem lentas.
```

File: packages/tsconfig-reference/copy/pt/options/generateCpuProfile.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+Essa opção só pode ser utilizada no CLI pelo comando: `--generateCpuProfile tsc-output.cpuprofile`.
+
+```sh
+npm run tsc --generateCpuProfile tsc-output.cpuprofile
+```
+
+Esse arquivo pode ser aberto em um navegador com base chromium, como o Chrome ou Microsoft Edge na seção [de perfil de CPU](https://developers.google.com/web/tools/chrome-devtools/rendering-tools/js-execution). Para saber mais sobre o desempenho dos compiladores, pode visitar a [wiki do TypeScript sobre performance](https://github.com/microsoft/TypeScript/wiki/Performance).
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/generateCpuProfile.md]:
```suggestion
Esse arquivo pode ser aberto em um navegador baseado no Chromium, como o Chrome ou Microsoft Edge na seção [de perfil de CPU](https://developers.google.com/web/tools/chrome-devtools/rendering-tools/js-execution). Para saber mais sobre o desempenho dos compiladores, pode visitar a [wiki do TypeScript sobre performance](https://github.com/microsoft/TypeScript/wiki/Performance).
```

File: packages/tsconfig-reference/copy/pt/options/importHelpers.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
+---
+display: "Importação de Auxiliares"
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/importHelpers.md]:
```suggestion
display: "Importar Auxiliares"
```

File: packages/tsconfig-reference/copy/pt/options/importHelpers.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+oneline: "Permite importar funções auxiliares por projeto, ao invés de incluir em arquivos individuais."
+
+---
+Para algumas operações de nível mais baixo, o TypeScript pode usar alguns códigos auxiliares para operações como extensão de classes, espalhar arrays ou objetos e para operações async. Por padrão esses auxiliares são inseridos em cada arquivo utilizado. Isso pode provocar duplicação de códigos se o mesmo auxiliar for usado em diferentes módulos.
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/importHelpers.md]:
```suggestion
---

Para algumas operações de mais baixo nível, o TypeScript pode usar alguns códigos auxiliares para operações como extensão de classes, fazer spread de arrays ou objetos e para operações async. Por padrão esses auxiliares são inseridos em cada arquivo utilizado. Isso pode provocar duplicação de códigos se o mesmo auxiliar for usado em diferentes módulos.
```

File: packages/tsconfig-reference/copy/pt/options/importHelpers.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+Se o `importHelpers` estiver ligado, as funções auxiliares serão importadas do módulo [tslib](https://www.npmjs.com/package/tslib).Você precisa ter certeza que o módulo `tslib` pode ser importado no runtime. Isso afeta apenas módulos; Os arquivos de scripts globais não tentarão importar módulos.
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/importHelpers.md]:
```suggestion
Se `importHelpers` estiver ligado, as funções auxiliares serão importadas do módulo [tslib](https://www.npmjs.com/package/tslib). Você precisa ter certeza que o módulo `tslib` pode ser importado no runtime. Isso afeta apenas módulos; Os arquivos de scripts globais não tentarão importar módulos.
```

**Review by khaosdoctor — CHANGES_REQUESTED** [review]

**Comment by juninhogomes** [comment]
Did the corrections suggested by @khaosdoctor. If there are others, just send me it! :smiley:

**Review by khaosdoctor — APPROVED** [review]
lgtm

**Comment by khaosdoctor** [comment]
lgtm

**Pull request merged**